daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -809,6 +817,7 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
                 SEVar(3), // choice argument
                 SEVar(4), // cid
                 SEVar(5), // actors
+                SEVar(6), // byKey flag
```

Comment:
Please add the `byKey` flag to the various `// stack:` comments.

Reply:
Good catch